src/test/kotlin/ControllerTest.kt

```diff
@@ -96,6 +97,33 @@ class ControllerTest {
         }
     }
 
+    @Test
```

Comment:
Burde vi ha to tester til, en med stillingskategori stilling, og en med stillingskategori null? Jeg antr det blir duplikat av innhold siden de andre testene sikkert har med kategori stilling, men kan være fint å ha test av alle disse tre permutasjonene eksplisitt navngitt og sammen.